### PR TITLE
remove experimental announcement from cloud audit

### PR DESCRIPTION
**Pull request by songrijie**
we remove experimental announcement as this feature has been verified by use cases.

### What is changed, added or deleted? (Required)

Experimental announcement is removed.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by ljun0712 — APPROVED** [review]

File: tidb-cloud/tidb-cloud-auditing.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ summary: Learn about how to audit a cluster in TiDB Cloud.
 
 # Database Audit Logging
 
-TiDB Cloud provides you with a database audit logging feature to record a history of user access details (such as any SQL statements executed) in logs.
-
-> **Note:**
+> **Note**
 >
-> Currently, the **audit logging** feature is experimental. The interface and output are subject to change.
+> Database Audit Logging has not been opened for all regions. Please log a support ticket if you don't have the setting in console.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-auditing.md:10]:
```suggestion
> The database audit logging feature has not been opened for all regions. If you want to use this feature but do not see **Audit Settings** in the TiDB Cloud console, you can contact the [TiDB Cloud support team](/tidb-cloud/tidb-cloud-support.md) for help.
```

**Comment by songrijie** [comment]
@qiancai Thanks for the suggestion, LGTM.

File: tidb-cloud/tidb-cloud-auditing.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ summary: Learn about how to audit a cluster in TiDB Cloud.
 
 # Database Audit Logging
 
-TiDB Cloud provides you with a database audit logging feature to record a history of user access details (such as any SQL statements executed) in logs.
-
-> **Note:**
+> **Note**
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-auditing.md:8]:
```suggestion
> **Note：**
```

File: tidb-cloud/tidb-cloud-auditing.md [hunk 3]
```diff
@@ -5,11 +5,11 @@ summary: Learn about how to audit a cluster in TiDB Cloud.
 
 # Database Audit Logging
 
-TiDB Cloud provides you with a database audit logging feature to record a history of user access details (such as any SQL statements executed) in logs.
-
 > **Note:**
 >
-> Currently, the **audit logging** feature is experimental. The interface and output are subject to change.
+> The database audit logging feature has not been opened for all regions. If you want to use this feature but do not see **Audit Settings** in the TiDB Cloud console, you can contact the [TiDB Cloud support team](/tidb-cloud/tidb-cloud-support.md) for help.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-auditing.md:10]:
```suggestion
> The database audit logging feature is disabled by default. If you want to enable this feature, contact the [TiDB Cloud support team](/tidb-cloud/[tidb-cloud-support.md](http://tidb-cloud-support.md/)) for help.
```

**Comment by qiancai** [comment]
The similar doc changes had been done in https://github.com/pingcap/docs/pull/12253, so this PR can be closed.

**Pull request closed without merging**